defaults/main.yml

```diff
@@ -85,24 +69,11 @@ qdr_listener_port: 5666
 
 qdr_log_output: /var/log/qdrouterd/metrics_qdr.log
 
-# Corresponds to MetricsQdrLoggingSource in THT
-metrics_qdr_logging_source: {'startmsg.regex': '^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}(.[0-9]+ \\+[0-9]+)? [A-Z]+ \\([a-z]+\\) ', 'tag': 'openstack.nova.consoleauth', 'file': '/var/log/containers/qdrouterd/metrics_qdr.log'}
```

Comment:
Unused (at least by this code) - there is code in THT for this still. It's used by rsyslog